plugin/trino-hive/src/test/java/io/trino/plugin/hive/TestHiveConfig.java

```diff
@@ -190,6 +191,7 @@ public void testExplicitPropertyMappings()
                 .put("hive.iceberg-catalog-name", "iceberg")
                 .put("hive.size-based-split-weights-enabled", "false")
                 .put("hive.minimum-assigned-split-weight", "1.0")
+                .put("hive.managed-table-auto-purge-default", "true")
```

Comment:
- we don't need "default". It's normal that config property can be overridden by table property (where applicable), so we don't need to emphasize this here
- the auto_purge table property is applicable to managed tables only, but it's name doesn't convey that. I think it's fine to call the catalog config property similarly: remove "managed-table-".

```suggestion
                .put("hive.auto-purge", "true")
```

Reply:
@electrum please review config property name